exe/kubernetes-run

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require 'kubernetes-deploy'
+require 'kubernetes-deploy/runner_task'
+require 'optparse'
+
+template = "task-runner-template"
+command = nil
+env_vars = []
+
+ARGV.options do |opts|
+  opts.on("--template=TEMPLATE") { |n| template = n }
+  opts.on("--env-vars=ENV_VARS") { |vars| env_vars = n.split(",")}
+  opts.on("--command=COMMAND") { |c| command = [c] }
+  opts.parse!
+end
+
+runner = KubernetesDeploy::RunnerTask.new(
+  namespace: ARGV[0],
+  context: ARGV[1],
+)
+
+ KubernetesDeploy::Runner.with_friendly_errors do
+  runner.run(
+    task_template: template,
+    command: command,
+    args: ARGV[2..-1],
```

Comment:
I call it command because thats what kubernetes calls it. Though I'm sure that you could make a container that accepts `rake` as a valid `command` parameter, I'll adjust it to be called entrypoint instead.